cpp/dolfinx/geometry/BoundingBoxTree.cpp

```diff
@@ -34,8 +34,8 @@ std::vector<std::int32_t> range(const mesh::Mesh& mesh, int tdim)
 }
 //-----------------------------------------------------------------------------
 // Compute bounding box of mesh entity
```

Comment:
Happy with the following
```suggestion
// Compute bounding box of mesh entity. The bounding box is defined by (lower left corner, top right corner).
// Storage flattened row-major
```
@IgorBaratta